default option is to build qt without STL support...
- Sebastian Czech <t2_@arcor.de>

--- hydrogen-0.9.1/hydrogen_gui/PreferencesDialog.cpp.orig	2004-11-28 16:09:19.000000000 +0100
+++ hydrogen-0.9.1/hydrogen_gui/PreferencesDialog.cpp	2005-03-27 13:42:41.000000000 +0200
@@ -84,7 +84,7 @@
 
 	// ALSA
 	m_pAlsaBufferSizeTxt->setValue( pPreferencesMng->getAlsaBufferSize() );
-	m_pAlsaDeviceEdit->setText( pPreferencesMng->getAlsaAudioDevice() );
+	m_pAlsaDeviceEdit->setText( pPreferencesMng->getAlsaAudioDevice().c_str() );
 	switch ( pPreferencesMng->getAlsaAudioDeviceSampleRate() ) {
 		case 44100:
 			m_pAlsaSampleRateComboBox->setCurrentItem( 0 );
